### PR TITLE
Tweaked src/battle_debug.c to comply with the PE

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
Due to the removal of the Deoxys hack in the pokemon_expansion branch, this function needed some minor adjustments to work on a merged branch. Adjustments that could have always been handled by using the #ifdef preproc directive.
This PR addresses that, thus giving the user one less thing to think about when merging the feature branches hosted in this repository manually.

## **Discord contact info**
Lunos#4026

### CODE REVIEW

**Comment by ultima-soul** [comment]
I'm not a fan of this because this is a really easy merge conflict to resolve.

**Comment by LOuroboros** [comment]
> 
> 
> I'm not a fan of this because this is a really easy merge conflict to resolve.

Huh. I see it as a pointless annoyance that users shouldn't need to go through.

File: src/battle_debug.c [hunk 1]
```diff
@@ -757,11 +763,19 @@ static void Task_ShowAiPoints(u8 taskId)
...
                                                  gBattleMons[data->aiBattlerId].personality,
                                                  TRUE,
                                                  39, 130, 15, 0xFFFF);
+    #else
+        data->aiMonSpriteId = CreateMonPicSprite(gBattleMons[data->aiBattlerId].species,
+                                                 gBattleMons[data->aiBattlerId].otId,
+                                                 gBattleMons[data->aiBattlerId].personality,
+                                                 TRUE,
+                                                 39, 130, 15, 0xFFFF);
+    #endif
```
Comment (ultima-soul) [inline src/battle_debug.c:778]:
```suggestion
        #ifndef POKEMON_EXPANSION
        data->aiMonSpriteId = CreateMonPicSprite_HandleDeoxys(gBattleMons[data->aiBattlerId].species,
                                                 gBattleMons[data->aiBattlerId].otId,
                                                 gBattleMons[data->aiBattlerId].personality,
                                                 TRUE,
                                                 39, 130, 15, 0xFFFF);
        #else
        data->aiMonSpriteId = CreateMonPicSprite(gBattleMons[data->aiBattlerId].species,
                                                 gBattleMons[data->aiBattlerId].otId,
                                                 gBattleMons[data->aiBattlerId].personality,
                                                 TRUE,
                                                 39, 130, 15, 0xFFFF);
        #endif
```

File: src/battle_debug.c [hunk 2]
```diff
@@ -866,11 +886,19 @@ static void Task_ShowAiKnowledge(u8 taskId)
...
```
Comment (ultima-soul) [inline src/battle_debug.c:901]:
```suggestion
        #ifndef POKEMON_EXPANSION
        data->aiMonSpriteId = CreateMonPicSprite_HandleDeoxys(gBattleMons[data->aiBattlerId].species,
                                                 gBattleMons[data->aiBattlerId].otId,
                                                 gBattleMons[data->aiBattlerId].personality,
                                                 TRUE,
                                                 39, 130, 15, 0xFFFF);
        #else
        data->aiMonSpriteId = CreateMonPicSprite(gBattleMons[data->aiBattlerId].species,
                                                 gBattleMons[data->aiBattlerId].otId,
                                                 gBattleMons[data->aiBattlerId].personality,
                                                 TRUE,
                                                 39, 130, 15, 0xFFFF);
        #endif
```

**Review by ultima-soul — CHANGES_REQUESTED** [review]

**Comment by LOuroboros** [comment]
I really don't like those changes tbqh with you, @ultima-soul, but done. Ready for re-review :eyes:

**Review by ultima-soul — APPROVED** [review]

**Pull request merged**